backend/routes_reports.py

```
@@ -355,3 +359,551 @@ def department_usage_report():
+                start_date = now - timedelta(days=90)
+            elif timeframe == 'year':
+                start_date = now - timedelta(days=365)
+            elif timeframe == 'all':
+                start_date = datetime(1970, 1, 1)
+            else:
+                start_date = now - timedelta(days=30)
+
+            # Get all discrepancy results within the timeframe
+            query = CycleCountResult.query.filter(
+                CycleCountResult.counted_at >= start_date,
+                CycleCountResult.has_discrepancy == True
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if CycleCountResult.has_discrepancy:` for truth checks

Replace with `CycleCountResult.has_discrepancy`

(E712)